changelog.txt

```diff
@@ -1,3 +1,23 @@
+14.1 (2024-05-03)
+------------------------------------------------------------------------
+Add: Check that towns can build roads before generating map (#12503)
+Fix #12228: CheckShipReverse only restricts path when it has to (#12474)
```

Comment:
this one feels like it could be improved. Something to do with ships ability to reverse, if I read the original issue/PR correctly?

Reply:
I've tried to improve it. Not sure if it's any better or correct.